### PR TITLE
Gerbera: update to 1.6.4

### PR DESCRIPTION
**Pull request by crater2150**
Updates Gerbera, also adds new package for libupnp1.12 (incompatible to versions 1.6 and 1.8 already in the repo)

### CODE REVIEW

**Comment by crater2150** [comment]
Crossbuilding libupnp1.12 to `x86_64-musl` and `i686` via `xbps-src -a` works without errors. Any ideas why it would fail in the CI build?

**Comment by tibequadorian** [comment]
libupnp1.12 is missing `CFLAGS="-D_FILE_OFFSET_BITS=64"`

**Comment by crater2150** [comment]
@tibequadorian Thanks, that fixed it.

The remaining linter warning concerns the `short_desc`, which I left consistent with `libupnp` and `libupnp1.8`.

**Comment by tibequadorian** [comment]
@crater2150 I would think it's better to make it consistently right than keep it consistently wrong ;) Its also already corrected in `libupnp1.8`

**Comment by crater2150** [comment]
Oh, it is indeed. changed it here too.

File: srcpkgs/libupnp1.12/template [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+post_install() {
+	vlicense COPYING
+}
+
+pre_configure() {
+	autoreconf -fi
+}
+
+libupnp1.12-devel_package() {
```
Comment (ericonr) [inline srcpkgs/libupnp1.12/template:25]: This should probably conflict with the other `libupnp*-devel` packages.

File: srcpkgs/libupnp1.12/template [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Template file for 'libupnp1.12'
+pkgname=libupnp1.12
+version=1.12.1
```
Comment (ericonr) [inline srcpkgs/libupnp1.12/template:3]: 1.14 exists... Does this really need 1.12? :(

Comment (crater2150) [reply srcpkgs/libupnp1.12/template:3]: 1.12 was the version listed in gerbera's readme. I'll try to build it against 1.14 and see if it works.

Comment (crater2150) [reply srcpkgs/libupnp1.12/template]: It compiles and seems to work with 1.14, so I replaced the 1.12 package.

**Comment by ahesford** [comment]
Related: #27829. The idea is to drop the current `libupnp` and, if possible, `libupnp1.8` (although this isn't a hard requirement) to reduce some of the UPnP madness going on here.

**Pull request closed without merging**